bitnami/nginx/values.yaml

```
@@ -300,7 +300,7 @@ pdb:
   minAvailable: 1
   ## @param pdb.maxUnavailable Max number of pods that can be unavailable after the eviction
   ##
-  maxUnavailable: 1
+  ## maxUnavailable: 1
```

We are transitioning to make every parameter of the `values.yaml` explicit:
```suggestion
  maxUnavailable:
```
In 97543ce I uncommented this parameter and inadvertently left the example value instead of leaving it `null` 🤦

---

> We are transitioning to make every parameter of the `values.yaml` explicit:

Oh sorry I have missed that!